WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/InitializationManager.java

```diff
@@ -71,6 +79,8 @@ private InitializationManager(Project project) {
         intelliJListener = new IntelliJListener(watchDogEventManager, project);
         intelliJListener.attachListeners();
         XDebuggerManager.getInstance(project).getBreakpointManager().addBreakpointListener(new BreakpointListener(eventManager));
```

Comment:
Why are these not in attachListeners? Perhaps rename the existing attachListeners, and create a new method attachDebugListeners?